forge-gui/res/cardsfolder/r/runesword.txt

```diff
@@ -2,11 +2,10 @@ Name:Runesword
 ManaCost:6
 Types:Artifact
 A:AB$ Pump | Cost$ 3 T | NumAtt$ +2 | ValidTgts$ Creature.attacking | TgtPrompt$ Select target attacking creature | SubAbility$ RuneswordEffect | SpellDescription$ Target attacking creature gets +2/+0 until end of turn. When that creature leaves the battlefield this turn, sacrifice CARDNAME. If the creature deals damage to a creature this turn, the creature dealt damage can't be regenerated this turn. If a creature dealt damage by the targeted creature would die this turn, exile that creature instead.
-SVar:RuneswordEffect:DB$ Effect | Name$ Runesword Effect | Triggers$ TrigSacRunesword,TrigNoregen | ReplacementEffects$ RuneswordRep | RememberObjects$ Targeted | ImprintCards$ Self
+SVar:RuneswordEffect:DB$ Effect | Name$ Runesword Effect | Triggers$ TrigSacRunesword | ReplacementEffects$ RuneswordRep | StaticAbilities$ NoRegen | RememberObjects$ Targeted | ImprintCards$ Self | ForgetOnMoved$ Battlefield | AILogic$ CantRegenerate
```

Comment:
this effect needs to stay even if the creature leaves?

Reply:
The Static and Replacement Effect probably want to have ForgetOnMoved

But the leave the Battlefield trigger interferes with it

can you help me to code the Leave The Battlefield trigger as Delayed Trigger? then it doesn't need to be part of the effect

Reply:
i moved the leave the battlefield trigger into a Delayed one

but i still removed ForgetOnMoved